docs/adding_packages/dependencies.md

```diff
@@ -178,14 +178,16 @@ for consumer, we do impose some limits on Conan features to provide a smoother f
 
 Version ranges are a useful Conan feature, [documentation here](https://docs.conan.io/2/tutorial/versioning/version_ranges.html).
 With the introduction of Conan 2.0, we are currently working to allow the use of version ranges and are allowing this for a handful of dependencies.
-Currently, these are:
+Currently, these are (except if the recipe needs a newer lower version for specific reasons):
 
 * OpenSSL: `[>=1.1 <4]` for libraries known to be compatible with OpenSSL 1.x and 3.x
 * CMake: `[>3.XX <4]`, where `3.XX` is the minimum version of CMake required by the relevant build scripts. Note that CCI recipes assume 3.15 is installed in the system, so add this
 version range only when a requirement for a newer version is needed.
-* Libcurl: `[>=X.YY <9]`, where `X.YY` is the minimum version of Libcurl required, starting from `7.78`
-* Zlib: `[>=1.2.11 <2]` expect if the recipe needs a newer lower version for specific reasons
-* Libpng: `[>=1.6 <2]` expect if the recipe needs a newer lower version for specific reasons
+* Libcurl: `[>=7.78 <9]`
+* Zlib: `[>=1.2.11 <2]`
+* Libpng: `[>=1.6 <2]`
+* Expat: `[>=2.6.2 <3]`
+* Libxml2: `[>=2.12.5 <3]`
```

Comment:
It there a rational for such high lower bound? I know that some recipes are not compatible with libxml2 2.12.x due to removal of few functions.

Reply:
Hi, this is just the version that had no cve at the time of writing

Note that when necessary, we can nudge the verison ranges to fit when a library does not support newer versions, I'll add a note, thanks for the heads up :)

Reply:
Pinging @SpaceIm in case you don't get answer notifications!